mineral-ui/mineral-ui
# [Website] Nav Guidelines

### Description

Refactor nav to enable easy addition of guidelines pages.
* Necessary in order to have place to add a common Render Props guidelines page in #735 
* Added new Import Syntax page (moved from /docs)
* Remove Palette Demo page

Also enables easy addition of Design/Design System specific content under the new Design heading.

### Motivation and context

closes #772 
connects #735 

### Screenshots, videos, or demo, if appropriate

https://772-nav-guidelines--mineral-ui.netlify.com/

### How to test

1. Verify that links work properly
2. Verify that content is appropriate

### Types of changes

- Other (provide details below)

Website navigation refactor

### Checklist
